app/src/main/java/com/hiddenramblings/tagmo/MainActivity.java

```diff
@@ -591,7 +592,9 @@ public void onDismissed(Snackbar snackbar, int event) {
             if (amiiboId == EditorTP.WOLF_LINK_ID) {
                 tpVisibility = View.VISIBLE;
             } else {
-                ssbVisibility = View.VISIBLE;
+                if ((amiiboId & AmiiboSeries.MASK)== 0x0000000000000000L){

```

Comment:
We should avoid using hard coded numbers. I suggest moving `0x0000000000000000L` to EditorSSB.java as a static variable (like `EditorTP.WOLF_LINK_ID` above).

Also change it to an `else if` :)